xclim/indicators/atmos/_temperature.py

```diff
@@ -215,6 +216,21 @@ class TempWithIndexing(ResamplingIndicatorWithIndexing):
     compute=indices.heat_wave_frequency,
 )
 
+heat_wave_magnitude = Temp(
+    title="Heat wave magnitude",
+    identifier="heat_wave_magnitude",
+    units="K d",
+    long_name="Maximum cumulative difference between daily maximum temperature and {thresh} for days within a heat wave. "
+    "A heat wave is defined as a series of at least {window} consecutive days with daily maximum temperature above "
+    "{thresh}.",
+    description="Magnitude of the most intensive heat wave per {freq}. A heat wave occurs when daily maximum "
+    "temperatures exceed {thresh} for {window} consecutive days.",
```

Comment:
I was just suggesting a bit more details 
```suggestion
 description="Magnitude of the most intensive heat wave per {freq}. The magnitude is the cumulative exceedance of daily maximum temperature over {thresh}. A heat wave is defined as a series of at least {window} consecutive days with daily maximum temperature above"
```